subsys/bluetooth/controller/ll_sw/ull_adv_sync.c

```diff
@@ -1818,3 +1815,26 @@ static void ticker_op_cb(uint32_t status, void *param)
 {
 	*((uint32_t volatile *)param) = status;
 }
+
+/**
+ * @brief Set a periodic advertising chained PDUs in a new periodic advertising data if there are
+ *        chained PDUs in former periodic advertising data.
+ *
+ * @param pdu_new  Pointer to new pdu_adv where to add chained pdu_adv.
+ * @param pdu_prev Pointer to former pdu_adv where to add chained pdu_adv.
+ */
+static void per_adv_data_set_chain_from_prev_pdu(struct pdu_adv *pdu_new, struct pdu_adv *pdu_prev)
```

Comment:
Do not use `per` as it is acronym for peripheral role in Bluetooth Test Spec, use `adv_sync` namespace as in other functions. `adv_sync_pdu_chain_check_and_duplicate` conveys the purpose of the function as it uses `adv_sync_pdu_duplicate_chain`

```suggestion
static void adv_sync_pdu_chain_check_and_duplicate(struct pdu_adv *pdu_new, struct pdu_adv *pdu_prev)
```

Reply:
The original comment in code go this way:
https://github.com/zephyrproject-rtos/zephyr/pull/43443/files#diff-3184590f25e7d9697815d4c8e4bc868dd3429067a409add63215421214f60eecL485-L489

new `PDU` having `chain` is `check`-ed, then `no need to deal with chain as it's already there`